src/pykeen/nn/node_piece.py

```diff
@@ -566,6 +566,103 @@ def resolve_aggregation(
     return aggregation
 
 
+class Tokenization(RepresentationModule):
```

Comment:
sweet insight: a tokenization is a representation module itself! (where the shape is prefixed by the number of selected tokens)